tests/tests/opendid.test.ts

```diff
@@ -0,0 +1,31 @@
+import { authentication } from './authentication'
+import { TestState } from './test_state'
+import { authorize } from './authorize'
+import { challenge } from './challenge'
+import { describe, it } from 'vitest'
+import { token } from './token'
+
+describe('OpenDID', async () => {
+  it('should support Authorization Code Flow', async () => {
+    const testState = new TestState()
```

Comment:
Can we make a before all and describe the test state inside so we don't repeat the teststate initialisation?